### ISSUE TITLE
"Locale" issue when building Windows Binaries with Docker and "non-git" repo source code

### ISSUE THREAD

**Issue by HardCorePawn** [spesmilo/electrum#6376]
As per my post on bitcointalk: https://bitcointalk.org/index.php?topic=5251565.msg54701595#msg54701595

I was attempting to assist another user who (for #reasons) wants to be able to build the Windows binaries _without_ needing to rely on git. They simply want to be able to download the source code in .tar.gz and build windows binaries.

As you can see from the linked post, if you attempt to simply download the .tar.gz from github and try to follow the build-wine instructions, the docker image build fails due to updated git version (not entirely unexpected because it was an old 3.3.8 package).

That's an easy enough fix, but after getting the docker image built, when we attempt to compile the Electrum binaries, the build script fails as there is no "git repo".

Manually creating a git repo, and adding the files etc, gets around that, but then we get to issues with the "locale" files. This error pops up in the middle:
![](https://i.imgur.com/24S1VMP.png)

and then the build process eventually dies here:
![](https://i.imgur.com/8Sj0tWq.png)

There seems to be some issue with the way it is retrieving and then extracting the "locale" files as it is creating a '*' folder instead of extracting the locale files to the correct subdirs??!?
![](https://i.imgur.com/P9UHy2C.png)

I tried manually downloading the locale files and then extracting them manually, but it still errors out.

NOTE: doing one simple "git clone ELECTRUM_REPO" makes all this go away and the build process works perfectly, but does not satisfy the users requirement to build without needing git.

Am I missing something obvious? 

TLDR: cannot build windows binaries from .tar.gz archive of Electrum repo, but git clone <ELECTRUM_REPO> allows building of windows binaries without error. help?
